addon/components/row/table-row.hbs

```diff
@@ -0,0 +1,39 @@
+<tr role="button" class="et-tr" {{on "click" this.onClick}} {{on "dblclick" this.onDoubleClick}}>
+    {{#each @api.cells as |api|}}
+        {{#if hasBlock}}
+            {{yield (hash
```

Comment:
even though its unlikely to be used here, I might keep the `isHeader` blocks from the original ember-tr component just for completeness. And would also make it easier to update and keep this component in sync with the base version if changes were ever made in the parent addon.